File: Java/Conectando.java
import java.sql.Connection;
import java.sql.DriverManager;

public class Conectando {

    public static void main(String[] args) {
        try {
            System.out.println("Conectando com o MySQL ...");
            System.out.println("Banco de Dados aula_bd");
            
            Class.forName("com.mysql.jdbc.Driver");
            String url = "jdbc:mysql://localhost:3306/aula_bd";
            String usuario = "root";
            String senha = "root";
            
            Connection mydb = DriverManager.getConnection(url, usuario, senha);
            System.out.println("Conectado? " + !mydb.isClosed());
        } catch (Exception e) {
            System.out.println("Erro: " + e);
        }
    }
}

File: Python/conectando.py
import mysql.connector

print('Conectando com o MySQL ...')
print('Banco de Dados aula_bd')

host = "localhost" #127.0.0.1
usuario = "root"
senha = "root"
banco = "aula_bd"

try:
    mydb = mysql.connector.connect(
        host=host,
        user=usuario,
        password=senha,
        database=banco
    )
    print("Conectado?", mydb.is_connected())    
except BaseException as e:
    print("Erro: ", e)
else:
    mydb.close()